ui/components/component-library/component-library-components.scss

```diff
@@ -5,11 +5,6 @@
 * unintended overrides.
 **/
 
-// Temp CSS Variables
-:root {
-  --opacity-disabled: 0.5;
-}
-
```

Comment:
Moved to `opacity.scss` as it's used outside of the component-library components now https://github.com/MetaMask/metamask-extension/pull/25158/files#diff-0bee63110d632972bcb15aeea3a018010055fb4c48040c989dfc7dbc329e4876